test/support/cli_helpers.ex

```diff
@@ -9,26 +9,46 @@ defmodule Support.CliHelpers do
   def run("cogctl" <> args) do
     capture_io(fn ->
       capture_io(:stderr, fn ->
-        try do
-          args
-          |> String.split
-          |> smart_split([])
-          |> Cogctl.main
-        catch
-          _, _ ->
-            nil
-        end
+        try_run("cogctl" <> args)
       end) |> IO.write
     end)
   end
   def run(_) do
     raise ~s(Commands must start with "cogctl")
   end
 
+  def run_capture_stderr("cogctl" <> args) do
+    capture_io(:stderr, fn ->
+      capture_io(fn ->
+        try_run("cogctl" <> args)
+      end)
+    end)
+  end
+
+  def run_capture_stdio("cogctl" <> args) do
+    capture_io(fn ->
+      capture_io(:stderr, fn ->
+        try_run("cogctl" <> args)
+      end)
+    end)
+  end
```

Comment:
Using `capture_io` twice in these two functions seems weird. What was the intention here? I think `run` captures both stdout and stderr but you should be able to just run `capture_io(:stderr, fn)` to capture only stderr.